base_layer/mmr/src/pruned_hashset.rs

```diff
@@ -80,6 +80,10 @@ impl ArrayLike for PrunedHashSet {
         Ok(self.base_offset + self.hashes.len())
     }
 
+    fn is_empty(&self) -> Result<bool, Self::Error> {
+        Ok(self.hashes.is_empty())
```

Comment:
I don't think this is correct as the hashes can be empty.
```suggestion
        Ok(self.len()==0)
```

Reply:
Alright, will change it back